### DISCUSSION TITLE
Adding id to markdown

### DISCUSSION THREAD

**Discussion by Alihtt** [agusmakmun/django-markdown-editor#264] [Q&A] [answered]
Hi I'm Ali and I've been coding for 2 years and I was using this module for my blog project and in this project I needed to have a markdown filed for my article description but the problem was the styling of that markdown so i added some extension to passing id to that markdown filed and the syntax is 

`# Hello world {hello_world_id}`

I would like to contribute and add this feature to this great and customizable module , please notify me if I can contribute this and add it.

**Comment by agusmakmun** [comment, marked as answer]
Hello @Alihtt, we have invited you as a contributor for this project.

![Screenshot 2024-11-11 at 09 25 27](https://github.com/user-attachments/assets/3d198b8a-4e0d-4a1c-9407-f23c6ff8adcf)

Thank you for willing to contribute.
